src/componet/Nav.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Nav = () => (
+  <nav className="nav bg-warning">
+    <ul className="d-flex justify-content-between w-100 my-2 mx-0">
+      <li className="fonts d-flex gap-2 list-unstyled">
+        <NavLink
+          className="text-decoration-none"
+          to="/"
+        >
+          <h3 className="fonts"> ≣ </h3>
+        </NavLink>
+        <h5 className="pl-3 mt-1 fonts pad">Crypto Coin Market Cap</h5>
+      </li>
+      <li className="d-flex list-unstyled px-0">
+        <h1>🎙</h1>
+        <h1 className="px-2 font-weight-bold">☼</h1>
+      </li>
+    </ul>
+  </nav>
```

Comment:
- [ ] Nice work on solving tests but the other issues pointed out by the previous reviewer are not yet addressed like:
- the video presentation needs to contain a highlighted code that you are proud of. kindly record a new video and make sure you have spoken about a piece of code you are proud of.
<img width="609" alt="image" src="https://user-images.githubusercontent.com/65068771/227475559-be43e943-6c97-4676-9115-1c46b0297b54.png">

- I have run your project locally and noticed that you don't have a back button, I can just see the home(hamburger)button on every page.
- Just like the previous reviewer said you need to follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
Typography: colors, font face, size, and weight
Layout: composition and space between elements.
<img width="577" alt="image" src="https://user-images.githubusercontent.com/65068771/227477646-59efc013-ead6-46bc-a646-4d52a09d6bf3.png">